src/main/java/org/folio/entlinks/service/messaging/authority/handler/UpdateAuthorityChangeHandler.java

```diff
@@ -67,12 +83,24 @@ public List<LinksChangeEvent> handle(List<AuthorityChangeHolder> changes) {
         linksEvents.addAll(handle0(change));
       } catch (AuthorityBatchProcessingException e) {
         log.warn("Skipping authority change processing.", e);
+        var report = new LinkUpdateReport();
+        report.setFailCause(e.getCause().getMessage());
+        report.setInstanceId(change.getAuthorityId());
+        report.setTenant(context.getTenantId());
+        report.setStatus(LinkUpdateReport.StatusEnum.FAIL);
+        report.setTs(DateUtils.fromTimestamp(Timestamp.from(Instant.now())).toString());
+        var producerRecord = new ProducerRecord<String, LinkUpdateReport>(topicName(), report);
+        linksUpdateKafkaTemplate.send(producerRecord);
```

Comment:
Forgot to add headers.
See `InstanceAuthorityLinkUpdateService` for `ProducerRecord` creation